[2025-09-25T11:25:43.087-0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-25T11:25:43.106-0300] {taskinstance.py:2631} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: WorkOrders.WorkOrders manual__2025-09-25T14:25:41.745321+00:00 [queued]>
[2025-09-25T11:25:43.112-0300] {taskinstance.py:2631} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: WorkOrders.WorkOrders manual__2025-09-25T14:25:41.745321+00:00 [queued]>
[2025-09-25T11:25:43.112-0300] {taskinstance.py:2884} INFO - Starting attempt 1 of 1
[2025-09-25T11:25:43.141-0300] {taskinstance.py:2907} INFO - Executing <Task(_PythonDecoratedOperator): WorkOrders> on 2025-09-25 14:25:41.745321+00:00
[2025-09-25T11:25:43.150-0300] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=262) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-09-25T11:25:43.152-0300] {standard_task_runner.py:72} INFO - Started process 264 to run task
[2025-09-25T11:25:43.151-0300] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'WorkOrders', 'WorkOrders', 'manual__2025-09-25T14:25:41.745321+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/Example.py', '--cfg-path', '/tmp/tmp43qjjn7z']
[2025-09-25T11:25:43.154-0300] {standard_task_runner.py:105} INFO - Job 30: Subtask WorkOrders
[2025-09-25T11:25:43.209-0300] {task_command.py:467} INFO - Running <TaskInstance: WorkOrders.WorkOrders manual__2025-09-25T14:25:41.745321+00:00 [running]> on host a44d6ec20546
[2025-09-25T11:25:43.293-0300] {taskinstance.py:3157} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='WorkOrders' AIRFLOW_CTX_TASK_ID='WorkOrders' AIRFLOW_CTX_EXECUTION_DATE='2025-09-25T14:25:41.745321+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-25T14:25:41.745321+00:00'
[2025-09-25T11:25:43.296-0300] {taskinstance.py:740} INFO - ::endgroup::
[2025-09-25T11:25:47.152-0300] {__init__.py:49} INFO - file_cache is only supported with oauth2client<4.0.0
[2025-09-25T11:25:48.602-0300] {logging_mixin.py:190} INFO - 0 novas linhas para inserir.
[2025-09-25T11:25:48.603-0300] {logging_mixin.py:190} INFO - Nada a inserir. Encerrando.
[2025-09-25T11:25:48.603-0300] {python.py:240} INFO - Done. Returned value was: None
[2025-09-25T11:25:48.614-0300] {taskinstance.py:349} INFO - ::group::Post task execution logs
[2025-09-25T11:25:48.615-0300] {taskinstance.py:361} INFO - Marking task as SUCCESS. dag_id=WorkOrders, task_id=WorkOrders, run_id=manual__2025-09-25T14:25:41.745321+00:00, execution_date=20250925T142541, start_date=20250925T142543, end_date=20250925T142548
[2025-09-25T11:25:48.664-0300] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-25T11:25:48.676-0300] {taskinstance.py:3924} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-25T11:25:48.678-0300] {local_task_job_runner.py:245} INFO - ::endgroup::
